start.py

```diff
@@ -230,6 +230,9 @@ def cm_server_not_dead(primary_node):
     # The work we need to do through CM itself begins here...
     deployment = ClouderaManagerDeployment(server_url)
     cm_cluster = deployment.cluster(DEFAULT_CLUSTER_NAME)
+    # For CDH 6, it takes a bit of time for CDH parcel to reach ACTIVATING or ACTIVATED stage.
+    # Hence wait for that stage to reach, otherwise the next statement to find cdh_parcel in one of those stages fails.
+    cm_cluster.wait_for_parcel_stage(product='CDH', version=args.cdh_version, stage='ACTIVATED')
     cdh_parcel = next(parcel for parcel in cm_cluster.parcels
                       if parcel.product == 'CDH' and parcel.stage in ('ACTIVATING',
                                                                       'ACTIVATED'))
```

Comment:
maybe should take out the check for: parcel.stage in ('ACTIVATING', ACTIVATED')? as the above statement already waited for it to be ACTIVATED.
otherwise this looks fine